### PR TITLE
CLD-2511: Support newer Uptycs TLS formats 2 and 3

### PR DESCRIPTION
**Pull request by uptycs-rmack**
To submit a PR please make sure to follow the next steps:

- [ ] Read the `CONTRIBUTING.md` guide at the root of the repo.
- [ ] Ensure the code is formatted building the `format_check` target.  
      If it is not, then move the committed files to the git staging area,
      build the `format` target to format them, and then re-commit.
      [More information is available on the wiki](https://osquery.readthedocs.io/en/latest/development/building/#formatting-the-code).
- [ ] Ensure your PR contains a single logical change.
- [ ] Ensure your PR contains tests for the changes you're submitting.
- [ ] Describe your changes with as much detail as you can.
- [ ] Link any issues this PR is related to.
- [ ] Remove the text above.

### CODE REVIEW

**Comment by spasam** [comment]
Crap, I force pushed the conversion warning fix and it looks like this diff got messed up

**Comment by uptycs-rmack** [comment]
OK, confirmed end-to-end with TLS from release branch. @spasam You want to approve? (And possibly make approvals mandatory)?

File: plugins/logger/tls_logger.cpp [hunk 1]
```diff
@@ -52,6 +52,18 @@ FLAG_ALIAS(google::uint64, logger_tls_max, logger_tls_max_linesize);
 
 FLAG(bool, logger_tls_compress, true, "GZip compress TLS/HTTPS request body");
 
+FLAG(uint64,
+     logger_tls_format,
+     1,
```
Comment (spasam) [inline plugins/logger/tls_logger.cpp:57]: Should we default to 3?

Comment (spasam) [reply plugins/logger/tls_logger.cpp:57]: Or is the CQ flags pushing down 3?

**Review by spasam — APPROVED** [review]

**Comment by uptycs-rmack** [comment]
Plan is to have config push down 3. That way old stack without new tls code
sets old format and new have new stack accepts new format can enable it.

**Pull request merged**